libraries/rush-lib/src/cli/RushXCommandLine.ts

```diff
@@ -206,24 +245,29 @@ export class RushXCommandLine {
     if (unknownArgs.length > 0) {
       // Future TODO: Instead of just displaying usage info, we could display a
       // specific error about the unknown flag the user tried to pass to rushx.
+      console.log(colors.red(`Unknown arguments: ${unknownArgs.map((x) => JSON.stringify(x)).join(', ')}`));
       help = true;
     }
 
     return {
       help,
       quiet,
+      isDebug,
+      ignoreHooks,
       commandName,
       commandArgs
     };
   }
 
   private static _showUsage(packageJson: IPackageJson, projectCommandSet: ProjectCommandSet): void {
     console.log('usage: rushx [-h]');
-    console.log('       rushx [-q/--quiet] <command> ...\n');
+    console.log('       rushx [-q/--quiet/--debug/--ignorehooks] <command> ...\n');
 
     console.log('Optional arguments:');
     console.log('  -h, --help            Show this help message and exit.');
-    console.log('  -q, --quiet           Hide rushx startup information.\n');
+    console.log('  -q, --quiet           Hide rushx startup information.');
+    console.log('  -ih, --ignorehooks    Do not run hooks.');
```

Comment:
Command-line short forms are designed so they can be chained into a single argument (i.e. `-qd` for `--quiet` + `--debug`). Having two characters in the short form will mess that up.

Reply:
@iclanton  Is `-ih` allowed? I thought `ts-command-line` bans that.

Reply:
Ohhh I see, `rushx` is not using `ts-command-line`.

Reply:
Well, let's do the right thing anyway.  What letter do you recommend?   -i  -k  ... something else?

Reply:
The easy answer is don't make a short parameter, and just call it `--ignore-hooks`.  Generally we only introduce short parameters for options that we believe will really be heavily used.  (Also, we strongly discourage short parameters in committed source code such as CI scripts, because it hinders readability for no good reason.  Short parameters should only be used by humans when typing in a shell prompt.)

If you do think this will be a very frequently used parameter, then:

I notice that `rush build` [already has](https://rushjs.io/pages/commands/rush_build/) an `--ignore-hooks` parameter.   It does not have a short version, but its short version ideally would be the same.  That means we should not use `-i` because `-i` already means `--impacted-by` in that context. And we can't use `-h` because that means `--help`.  So maybe `-g`?  That letter doesn't seem to be used by any existing parameters.

Reply:
To help this PR along, I've removed `-i`. Feel free to re-add `-g` if you feel strongly about it.

Reply:
Thanks.  I agree with that change.